python/tvm/contrib/binutil.py

```diff
@@ -45,25 +58,18 @@ def tvm_callback_get_section_size(binary_path, section_name, toolchain_prefix):
         size of the section in bytes
     """
     if not os.path.isfile(binary_path):
-        raise RuntimeError("no such file \"{}\"".format(binary_path))
+        raise RuntimeError('no such file \"{}\"'.format(binary_path))
```

Comment:
try to use double quote to be consistent with the rest part of the codebase

Reply:
Is that the convention?  I see single quotes and double quotes mixed in most parts of the codebase, but single quotes are a little bit easier to type.

Reply:
It is certainly bad to have a mixed style in the same file. There is no explicit mention of the force use of quote, but would be nice to have the same convention.